question-service/routes/question-route.js

```diff
@@ -1,9 +1,13 @@
 import express from 'express';
-import { getAllQuestions } from '../controller/question-controller.js'; // Use `import` and .js extension for ES modules
+import { getAllQuestions, fetchQuestionById, postQuestion } from '../controller/question-controller.js'; // Use `import` and .js extension for ES modules
 
 const router = express.Router();
 
 // Route to get all questions
 router.get('/', getAllQuestions);
 
+router.get("/:questionId", fetchQuestionById);
+
+router.post("/add", postQuestion);
```

Comment:
Nit: I think its better to have consistent naming across functions
Maybe `fetchQuestionById` -> `getQuestionById`